admin/templates/configuration/etc/init.d/mysql-proxy

```diff
@@ -78,6 +78,12 @@ if [ "$USER" = "root" ]; then
     proxy_user_option="--user=$proxy_user"
 fi
 
+# This parameter is required for achieving the most optimal performance of
+# the xrootd client. The parameter setting will be moved into a configuration file
+# of the proxy when it would be possible (there will be XrootD API for that) to set
+# a desired value of the parameter programmatically.
+export XRD_PARALLELEVTLOOP=100
```

Comment:
Maybe stick a TODO in here so it is less likely to be forgotten.

Reply:
Good suggestion! Same thought crossed my mind. I will do so.